code/modules/mob/living/carbon/human/species/station/vaurca/vaurca.dm

```diff
@@ -56,7 +56,7 @@
 	breath_eff_mul = 6 // 1/6 * breath_eff_mul = fraction of gas consumed
 	poison_type = GAS_NITROGEN //a species that breathes plasma shouldn't be poisoned by it.
 	breathing_sound = null //They don't work that way I guess? I'm a coder not a purple man.
-	mob_size = 13 //their half an inch thick exoskeleton and impressive height, plus all of their mechanical organs.
+	mob_size = 9 //their half an inch thick exoskeleton and impressive height, plus all of their mechanical organs.
```

Comment:
9 is the same as a normal humans, aren't they supposed to be bigger than a human?

Reply:
i'm aware it's the same size as normal humans, but the requirement to cause the quakejump means the mob size is 10 >. i could possibly put them at 10, but i'd have to test it